components/layout/variables.styl

```diff
@@ -98,7 +98,9 @@ privacy-light-color(prefix, attr)
   colors = 'closed' closed, 'public' public, 'private' private
   for value in colors
     {prefix + value[0]}
-      {attr} alpha(value[1], 0.1)
+      if attr == border-color
+        border 2px solid
+      {attr} alpha(value[1], 0.3)
```

Comment:
Was trying to avoid
```styl
.my-foo--private
  border-color borderColor(colors-channel-private)
.my-foo--public
  border-color borderColor(colors-channel-public)
.my-foo--closed
  border-color borderColor(colors-channel-closed)
```

But yeah, it's probably way easier to parse that way